[Bug]: TS type-check build issue

### Contact Details

jacktrute@letsloopin.com

### What happened?

Hey. I'm trying out Portkey within a Redwoodjs stack, but I get a TypeScript compile issue. I'm using `"portkey-ai": "1.3.0"`.

Not sure if I can avoid in via `tsconfig.json` or not. I'm originally from a TS background so might be missing something. I've tried adding `"skipLibCheck": true` but that didn't help.

### Version

0.1.xx (Default)

### Relevant log output

```shell
../node_modules/portkey-ai/dist/src/client.ts:9:11 - error TS2612: Property 'apiKey' will overwrite the base property in 'ApiClient'. If this is intentional, add an initializer. Otherwise, add a 'declare' modifier or remove the redundant declaration.

9  override apiKey: string | null;
            ~~~~~~

../node_modules/portkey-ai/dist/src/client.ts:10:11 - error TS2612: Property 'baseURL' will overwrite the base property in 'ApiClient'. If this is intentional, add an initializer. Otherwise, add a 'declare' modifier or remove the redundant declaration.

10  override baseURL: string;
             ~~~~~~~


Found 2 errors in the same file, starting at: ../node_modules/portkey-ai/dist/src/client.ts:9
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/src/client.ts
+++ b/src/client.ts
@@ -6,8 +6,8 @@ import { MISSING_API_KEY_ERROR_MESSAGE, PORTKEY_BASE_URL } from "./constants";
 import { castToError, readEnv } from "./utils";
 
 export class Portkey extends ApiClient {
-	override apiKey: string | null;
-	override baseURL: string;
+	declare apiKey: string | null;
+	declare baseURL: string;
 	virtualKey: string | null;
 	config: Record<string, unknown> | string | null | undefined;
 	provider: string | null | undefined;

